### DISCUSSION TITLE
Gratuluje ... CERT Was promuje 🗡️

### DISCUSSION THREAD

**Discussion by MajkiIT** [FiltersHeroes/KAD#1831] [Pokaż i opowiedz]
![obraz](https://user-images.githubusercontent.com/11302675/125645019-4798918f-0c68-4e7f-baa1-6de271e8b3a9.png)

https://youtu.be/3o4C4D-0zKg?t=874

**Comment by hawkeye116477** [comment]
No to super...

PS: My tu gadu gadu, a tymczasem czekają na @MajkiIT zaległe sprawy jak https://github.com/MajkiIT/polish-ads-filter/issues/19917 :smile:
